pkg/app/app.go

```diff
@@ -0,0 +1,6 @@
+package app
+
+// App is the model struct for apps
+type App struct {
+	ID string `json:"id"`
```

Comment:
use the swagger api docs populate this section chrome-extension://ljlmonadebogfjabhkppkoohjkjclfai/index.html?url=https://raw.githubusercontent.com/aerogear/mobile-security-service/master/apispec.yaml#!/apps/getApps